python/modules/xml_parser.py

```diff
@@ -35,18 +35,49 @@ def get_reference(self, env): # function name requires changes
     def get_value(self, env, args = {}):
         return env[self.value].get_value(env, args)
 
-def parse_param(param_dict, param):
+class VariableList():
+    def __init__(self, key, value, args = {}):
+        Variable.__init__(self, key, value, args = {})
+        # not sure if this should be subclass
+
+    def get_value(self, env, args = {}):
+        order = sorted(self.value.keys())
+        values_list = []
+        for key in order:
+            v_part = self.value[key].get_value(env)
+            #print(v_part)
+            values_list.append(v_part)
+        return "".join(values_list)
+
+def _parse_parted_param(param_dict, param, parted_dict, p_part):
+    key = param.get('key')
+    if key not in parted_dict:
+        parted_dict[key] = {}
+    variable = _create_variable(param)[0]
+    parted_dict[key][p_part] = variable
+
+def _add_variable_list(settings, parted_params):
+    for key, params_set in parted_params.items():
+        variable = VariableList(key, parted_params[key])
+        settings[key] = variable
+
+def _create_variable(param):
     key = param.get('key')
     value = param.get('value')
-    try:
-        p_type = param.get('type')
-    except:
-        p_type = "string"
+    p_type = param.get('type')
```

Comment:
Because I noticed that this 'try' will always execute, in case of no "type" instance in param it will just return "None".